content/blog/2022-05-24-MLEM-release.md

```diff
@@ -0,0 +1,277 @@
+(AWS Sagemaker and Kubernetes are coming).
+
+```shell
+$ mlem deploy create myservice -m mlem-model -t staging -c app_name=mlem-quick-start
+⏳️ Loading deployment from my-service.mlem
+🔗 Loading link to staging.mlem
+🔗 Loading link to mlem-model.mlem
+💾 Updating deployment at my-service.mlem
+🛠 Creating docker image for heroku
+  🛠 Building MLEM wheel file...
+  💼 Adding model files...
+  🛠 Generating dockerfile...
+  💼 Adding sources...
+  💼 Generating requirements file...
+  🛠 Building docker image registry.heroku.com/mlem-quick-start/web...
+  ✅  Built docker image registry.heroku.com/mlem-quick-start/web
+  🔼 Pushing image registry.heroku.com/mlem-quick-start/web to registry.heroku.com
+  ✅  Pushed image registry.heroku.com/mlem-quick-start/web to registry.heroku.com
+💾 Updating deployment at my-service.mlem
+🛠 Releasing app mlem-quick-start formation
+💾 Updating deployment at my-service.mlem
+✅  Service mlem-quick-start is up. You can check it out at https://mlem-quick-start.herokuapp.com/
+```
+
+# Fast model registry setup based on Git
+
+[versions]: /doc/use-cases/versioning-data-and-model-files
+[mp]: /doc/start/metrics-parameters-plots
+[experiments]: /doc/user-guide/experiment-management
+[gto]: https://github.com/iterative/gto
+[mlem]: https://mlem.ai/
+[modeling process]: /doc/start/data-pipelines
+[remote storage]: /doc/command-reference/remote
+[sharing]: /doc/start/data-and-model-access
+[via cml]: https://cml.dev/doc/cml-with-dvc
+[gitops]: https://www.gitops.tech/
+
+MLEM is a core building block for a Git-based ML model registry, together with
+other Iterative tools, like GTO and DVC.
+
+ML model registries give your team key capabilities:
+
+- Collect and organize model [versions] from different sources effectively,
+  preserving their data provenance and lineage information.
+- Share metadata including [metrics and plots][mp] to help use and evaluate
+  models.
+- A standard interface to access all your ML artifacts, from early-stage
+  [experiments] to production-ready models.
+- Deploy specific models on different environments (dev, shadow, prod, etc.)
+  without touching the applications that consume them.
+- For security, control who can manage models, and audit their usage trails.
+
+Many of these benefits are built into DVC: Your [modeling process] and
+[performance data][mp] become **codified** in Git-based <abbr>DVC
+repositories</abbr>, making it possible to reproduce and manage models with
+standard Git workflows (along with code). Large model files are stored
+separately and efficiently, and can be pushed to [remote storage] -- a scalable
+access point for [sharing].
+
+To make a Git-native registry (on top of DVC or not), one option is to use [GTO]
+(Git Tag Ops). It tags ML model releases and promotions, and links them to
+artifacts in the repo using versioned annotations. This creates abstractions for
+your models, which lets you **manage their lifecycle** freely and directly from
+Git.
```

Comment:
This part I took from @jorgeorpinel's PR: https://github.com/iterative/dvc.org/pull/3333

Reply:
If you want to reuse explanations from other places that's fine but rephrase them in your own words (the way you understand it). Blog posts should have the author's voice IMO.

OK to have very small sections (ad admonition, a sentence or 2) copy/pasted between blog and docs.